File: algo-c-to-_/examples/isbn13_.py
#
#	from src/isbn13.c
#
#	a part of main		to	isISBN13
#

from checkdigit import isISBN13

def p(s):
    print("{0} is {1}.".format(s, isISBN13(s) and "valid" or "invalid"))

p("9784774196909")

File: porting-from-algo-c/examples/isbn13.lua
--
--	from src/isbn13.c
--
--	a part of main		to	isIsbn13
--

local isISBN13 = require 'checkdigit'.isISBN13

do
	local s = "9784774196909"
	print(("%s is %s."):format(s, isISBN13(s) and "valid" or "invalid"))
end
